README.md

```diff
@@ -1 +1,110 @@
-# JS_Capastone
\ No newline at end of file
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet) ![HTML](https://img.shields.io/badge/-HTML-orange) ![CSS](https://img.shields.io/badge/-CSS-blue) ![JavaScript](https://img.shields.io/badge/-JavaScript-yellow)
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly add the correct [readme file](https://github.com/microverseinc/readme-template) and customize it accordingly. 
- [ ] After adding the correct readme file you remove the second license and maintain one license.